packages/waku/src/router/create-pages.tsx

```diff
@@ -352,26 +354,10 @@ export const createPages = <
     }
   };
 
-  // FIXME this should be refactored
```

Comment:
Thanks for working on it. 
What I wondered is that we don't need createRouterRenders in the first place.
In particular, 
```ts
        if (!renderer) {
          throw new Error('Renderer not found for ' + id);
        }
```
shouldn't be required. (I don't mean to just remove it, but I see if we restructure our logic, we don't even need it.)

Reply:
it looks good.